lib/blueprinter/serializers/association_serializer.rb

```diff
@@ -1,10 +1,8 @@
 class Blueprinter::AssociationSerializer < Blueprinter::Serializer
   def serialize(association_name, object, local_options, options={})
-    if options[:blueprint]
-      view = options[:view] || :default
-      options[:blueprint].prepare(object.public_send(association_name), view_name: view, local_options: local_options)
-    else
-      object.public_send(association_name)
-    end
+    value = object.public_send(association_name)
+    return value if value.nil?
+    view = options[:view] || :default
+    options[:blueprint].prepare(value, view_name: view, local_options: local_options)
```

Comment:
So, I'm not sure if you intended this or not, but a side effect of this code is that it makes it so that you now always *have* to have a Blueprint for every association. I think that could be considered a reasonable tradeoff, but I want to make sure we're explicitly making that decision.

Reply:
Ya, so I do not understand how it would work if the associated object does not have a blueprint.

Reply:
I am for having the requirement that any object being serialized within this system must use a blueprint... I've personally been bitten by having both AMS and jbuilder for the same resource far too many times to try to fashion some coexistence between any of them.